scipy/stats/_morestats.py

```diff
@@ -3094,24 +3107,35 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
 
     Notes
     -----
-    The test has been introduced in [4]_. Given n independent samples
-    (xi, yi) from a bivariate distribution (i.e. paired samples),
-    it computes the differences di = xi - yi. One assumption of the test
-    is that the differences are symmetric, see [2]_.
-    The two-sided test has the null hypothesis that the median of the
-    differences is zero against the alternative that it is different from
-    zero. The one-sided test has the null hypothesis that the median is
-    positive against the alternative that it is negative
-    (``alternative == 'less'``), or vice versa (``alternative == 'greater.'``).
-
-    To derive the p-value, the exact distribution (``method == 'exact'``)
-    can be used for small sample sizes. The default ``method == 'auto'``
-    uses the exact distribution if there are at most 50 observations and no
-    ties, otherwise a normal approximation is used (``method == 'approx'``).
-
-    The treatment of ties can be controlled by the parameter `zero_method`.
-    If ``zero_method == 'pratt'``, the normal approximation is adjusted as in
-    [5]_. A typical rule is to require that n > 20 ([2]_, p. 383).
```

Comment:
Moved this elsewhere.

Reply:
I would just say that the part "Given n independent samples (xi, yi) from a bivariate distribution (i.e. paired samples)" explained the underlying assumptions a bit better than the new version below...

Maybe you can write the following in your suggestion below: 
"In the following, let ``d`` represent the difference between the paired samples x and y (i.e., (x1, y1), ...(xn, yn) are assumed to be independent samples)"